src/js/admin-edit-profile.js

```diff
@@ -175,6 +179,142 @@ var wsuwp = wsuwp || {};
 			$( "[data-for='" + field + "']" ).eq( index ).remove();
 			$( this ).remove();
 		} );
+
+		// Surface photo collection.
+		$card.on( "click", ".photo", function() {
+			if ( $( this ).hasClass( "wsu-person-add-photo" ) ) {
+				return;
+			}
+
+			var position = $( this ).offset();
+			previous_focus = document.activeElement;
+
+			$( window ).scrollTop( 0 );
+			$( "body" ).addClass( "wsu-person-photo-collection-open" );
+
+			$photo_collection.css( {
+				"top": position.top,
+				"left": position.left
+			} );
+
+			$( ".wsu-person-add-photo" ).focus();
+		} );
+
+		// Handle keyboard interactions with the photo collection.
+		$( document ).keydown( function( e ) {
+			if ( !$( "body" ).hasClass( "wsu-person-photo-collection-open" ) ) {
+				return;
+			}
+
+			// Trap the focus within the photo collection.
```

Comment:
I had to lookup that 9 = "tab". That's probably uncommon enough to note in the inline comment.